Gaijinizer patch/Map218.txt

```diff
@@ -447,16 +476,18 @@ human weapons are not enough#MTLed
 いくら肉体や精神が悪魔との戦いに慣れていても
 脆い人間の肉体だと言うことを忘れるなよ。
 > CONTEXT: Map218/events/1/pages/0/211/Dialogue/other5
-Rather, I am worried about you.
-No matter how much your body and spirit are used
-to fighting demon. Don't forget to say that it is
-a fragile human body.#MTLed
+Rather, it's you I'm worried about. No matter how
+used your body and mind becomes to battling
+demons, don't forget that your body is that of a
+fragile human.#RonIffyWording
 > END STRING
```

Comment:
minor rewording:
>_**Rather, it's you I'm worried about. No matter how
good your body and mind becomes at battling
demons, don't forget it's still that of a fragile
human.**_